### PR TITLE
style: minor UI fixes

### PR DESCRIPTION
**Pull request by United600**
- Adjust custom accent brush to align with WinUI
- Fix `AppTitleBar` title inactive brush on theme switching
- Fix, and adjust, `PlayerPage` `Header` and `PlayerControls` background for the light theme
- Remove some `XamlUICommand`
- Fix "squashed" `ShuffleAndPlay` button in the `ArtistsPage` minimal visual state
- Fix potential overlap between `BreadcrumbBar` and `NoNetworkDrivePanel` on `NetworkPage`
- Add `ScrollViewer.Is*ScrollChainingEnabled` to `SemanticZoom` control according to the [documentation](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.semanticzoom?view=winrt-26100)

### CODE REVIEW

File: Screenbox/Pages/HomePage.xaml [hunk 1]
```diff
@@ -23,37 +23,29 @@
...
-                x:Key="RemoveCommand"
-                Command="{x:Bind ViewModel.RemoveCommand}"
-                IconSource="{ui:SymbolIconSource Symbol=Clear}"
-                Label="{strings:Resources Key=Remove}" />
-
-            <XamlUICommand
-                x:Key="PropertiesCommand"
-                Command="{StaticResource ShowPropertiesCommand}"
-                IconSource="{ui:FontIconSource Glyph=&#xE946;}"
-                Label="{strings:Resources Key=Properties}" />
```
Comment (huynhsontung) [inline Screenbox/Pages/HomePage.xaml:49]: To give some context, the XamlUICommands are there to be shared between context menu items and future command bar buttons, like the inbox media player. It's fine to remove these for now.

File: Screenbox/Pages/AlbumsPage.xaml [hunk 2]
```diff
@@ -75,8 +75,14 @@
...
             HorizontalAlignment="Right"
             AccessKey="{strings:KeyboardResources Key=CommandSortByKey}"
+            AutomationProperties.Name="{x:Bind SortByText.Text}"
             Background="Transparent"
             BorderBrush="Transparent">
+            <StackPanel Orientation="Horizontal">
+                <FontIcon Margin="0,0,8,0" Glyph="&#xE8CB;" />
+                <TextBlock x:Name="SortByText"><Run Text="{strings:Resources Key=SortBy}" /><Run Text=":&#160;" /></TextBlock>
+                <TextBlock Foreground="{ThemeResource AccentTextFillColorPrimaryBrush}" Text="{x:Bind GetSortByText(ViewModel.SortBy), Mode=OneWay}" />
+            </StackPanel>
```
Comment (huynhsontung) [inline Screenbox/Pages/AlbumsPage.xaml]: I always put element properties first then child elements. Please follow this convention in all XAML files.

File: Screenbox/Pages/MainPage.xaml.cs [hunk 3]
```diff
@@ -81,19 +81,18 @@ private void CoreTitleBar_LayoutMetricsChanged(CoreApplicationViewTitleBar sende
...
-            SolidColorBrush defaultForegroundBrush = (SolidColorBrush)Application.Current.Resources["TextFillColorPrimaryBrush"];
-            SolidColorBrush inactiveForegroundBrush = (SolidColorBrush)Application.Current.Resources["TextFillColorDisabledBrush"];
-
             if (args.WindowActivationState == CoreWindowActivationState.Deactivated)
             {
-                AppTitleText.Foreground = inactiveForegroundBrush;
+                VisualStateManager.GoToState(this, "Deactivated", true);
             }
-            else
+            else if (args.WindowActivationState == CoreWindowActivationState.PointerActivated || args.WindowActivationState == CoreWindowActivationState.CodeActivated)
```
Comment (huynhsontung) [inline Screenbox/Pages/MainPage.xaml.cs]: Why do we need this extra condition? `CoreWindowActivationState` only has 3 possible values. An else clause should be sufficient.

File: Screenbox/Pages/SongsPage.xaml [hunk 4]
```diff
@@ -26,30 +26,24 @@
...
-                Command="{x:Bind ViewModel.PlayNextCommand}"
-                IconSource="{ui:FontIconSource FontFamily={StaticResource ScreenboxSymbolThemeFontFamily},
-                                               Glyph=&#xF5EB;}"
-                Label="{strings:Resources Key=PlayNext}" />
-
-            <XamlUICommand
-                x:Key="PropertiesCommand"
-                Command="{StaticResource ShowPropertiesCommand}"
-                IconSource="{ui:FontIconSource Glyph=&#xE946;}"
-                Label="{strings:Resources Key=Properties}" />
```
Comment (huynhsontung) [inline Screenbox/Pages/SongsPage.xaml:46]: Same context as the MainPage

File: Screenbox/Pages/SongsPage.xaml [hunk 5]
```diff
@@ -94,8 +88,14 @@
...
             HorizontalAlignment="Right"
             AccessKey="{strings:KeyboardResources Key=CommandSortByKey}"
+            AutomationProperties.Name="{x:Bind SortByText.Text}"
             Background="Transparent"
             BorderBrush="Transparent">
+            <StackPanel Orientation="Horizontal">
+                <FontIcon Margin="0,0,8,0" Glyph="&#xE8CB;" />
+                <TextBlock x:Name="SortByText"><Run Text="{strings:Resources Key=SortBy}" /><Run Text=":&#160;" /></TextBlock>
+                <TextBlock Foreground="{ThemeResource AccentTextFillColorPrimaryBrush}" Text="{x:Bind GetSortByText(ViewModel.SortBy), Mode=OneWay}" />
+            </StackPanel>
```
Comment (huynhsontung) [inline Screenbox/Pages/SongsPage.xaml]: Please follow the convention and undo this change.

**Review by huynhsontung — CHANGES_REQUESTED** [review]

File: Screenbox/Pages/HomePage.xaml (hunk 1 above)
Comment (United600) [reply Screenbox/Pages/HomePage.xaml:49]: Didn't knew that, but wouldn't it be better for sharing to simply extract the entire MenuFlyout?

File: Screenbox/Pages/AlbumsPage.xaml (hunk 2 above)
Comment (United600) [reply Screenbox/Pages/AlbumsPage.xaml]: My bad, I consider both to be properties, and in my view, `Content` should almost always come first. I'll revert.

File: Screenbox/Pages/HomePage.xaml (hunk 1 above)
Comment (huynhsontung) [reply Screenbox/Pages/HomePage.xaml:49]: What do you mean by extracting the entire menu Flyout? XamlUICommand includes command, icon, label, and keyboard accelerator. Having controls share the same XamlUICommand will ensure they behave the same, functionally and aesthetically.

Comment (United600) [reply Screenbox/Pages/HomePage.xaml:49]: Like [this](https://github.com/videolan/vlc-winrt/blob/master/app/VLC.Universal/Views/UserControls/Flyouts/AlbumMenuFlyout.xaml)

Comment (huynhsontung) [reply Screenbox/Pages/HomePage.xaml:49]: I'm confused. How would you map these MenuFlyoutItems into Buttons or AppBarButtons?

Comment (United600) [reply Screenbox/Pages/HomePage.xaml:49]: Oh maybe [this](https://github.com/videolan/vlc-winrt/blob/master/app/VLC.Universal/Resources/Flyouts.xaml)? There must be a way

File: Screenbox/App.xaml [hunk 6]
```diff
@@ -18,10 +18,10 @@
                     <!--  Brushes  -->
                     <ResourceDictionary.ThemeDictionaries>
                         <ResourceDictionary x:Key="Default">
-                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight2" />
+                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight3" />
                         </ResourceDictionary>
                         <ResourceDictionary x:Key="Light">
-                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorDark1" />
+                            <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorDark2" />
```
Comment (huynhsontung) [inline Screenbox/App.xaml]: This resource only applies to Animated icons and thus causes color mismatch

image

Comment (huynhsontung) [reply Screenbox/App.xaml]: I will be reverting this so I can merge this PR. Please feel free to submit this change again in another PR.

**Review by huynhsontung — APPROVED** [review]
